framework/utilities/ecl-colorize/_colorize.scss

```diff
@@ -0,0 +1,38 @@
+/*
+  ECL Colorize
+  ===========
+*/
+
+@each $key, $value in $colors {
+  .ecl-bg-#{$key} {
+    background-color: $value;
+  }
+
+  .ecl-text-#{$key} {
+    color: $value;
+  }
+}
+
+.ecl-bg-primary { background-color: $ecl-primary; }
+.ecl-text-primary { color: $ecl-primary; }
```

Comment:
As mentioned earlier, `.ecl-color-***` might be more meaningful than `.ecl-text-***`